config/manager/manager.yaml

```diff
@@ -64,19 +62,64 @@ spec:
       serviceAccountName: controller-manager
       terminationGracePeriodSeconds: 10
 ---
+apiVersion: apps/v1
```

Comment:
let's pull this into a separate file, or even a separate folder tree for the flotta-api-server